### ISSUE TITLE
Cannot focus the Favorite button

### ISSUE THREAD

**Issue by mkruselj** [surge-synthesizer/surge#7399]
Navigating through the panes with focus debugger enabled, I cannot seem to be able to focus on the Favorite button in the Patch Selector. So we cannot right-click it either accessibly so that we get the list of favorites.

~~Additionally, it would be great if we had a search query that would list all favorites, like `CAT` or `CATEGORY = Fav` or `Favorites`.~~ (bp moved this to #7465)
